.github/workflows/build_and_upload_artifacts.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-12-xlarge]
+        os: [ubuntu-20.04, macos-12-xl]
```

Comment:
This is a x86 runner. The produced artifacts are no longer for arm now.

Reply:
Yes, I'll remove that, thanks.